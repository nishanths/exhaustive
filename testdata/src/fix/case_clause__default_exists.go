package fix

func _a() {
	var d Direction
	switch d { // want "^missing cases in switch of type Direction: E, directionInvalid$"
	case N:
	case S:
	case W:
		_ = 3
	default:
	}
}
