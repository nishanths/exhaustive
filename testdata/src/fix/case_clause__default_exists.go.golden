package fix

import (
	"fmt"
)

func _a() {
	var d Direction
	switch d { // want "^missing cases in switch of type Direction: E, directionInvalid$"
	case N:
	case S:
	case W:
		_ = 3
	case E, directionInvalid:
		panic(fmt.Sprintf("unhandled value: %v", d))
	default:
	}
}
